lib/nuntiux/mocker.ex

```diff
@@ -31,20 +60,7 @@ defmodule Nuntiux.Mocker do
            :reset_history
            | {:delete, expect_id()}
            | {:expect, expect_id(), expect_fun()}
-
-  @mocked_process_key :"#{__MODULE__}.mocked_process"
-  @default_history []
-  @default_opts [{:passthrough?, true}, {:history?, true}]
-  @default_expects %{}
-  @default_state %{
-    history: @default_history,
-    opts: @default_opts,
-    expects: @default_expects
-  }
-  @label_call :"$nuntiux.call"
-  @label_cast :"$nuntiux.cast"
-  @label_process_pid :"$nuntiux.process_pid"
-  @label_current_message :"$nuntiux.current_message"
+  @typep expects_matched :: unquote(@label_nomatch) | {unquote(@label_match), expect_fun_result()}
```

Comment:
Because I wanted to declare a type based on module attributes, while keeping some aesthetics to the code, I moved stuff around...